diff --git a/lib/parsers.js b/lib/parsers.js
index 3375f70ff7465318407a9ac04e95a865144d26ce..8659d62d4e5716a474d1ee59cea99aaaa00db123 100644
--- a/lib/parsers.js
+++ b/lib/parsers.js
@@ -2,11 +2,11 @@ var ParseAuth= /(\w+)\s+(.*)/  // -> scheme, params
   , Separators= /([",=])/
   ;
 
-parse_params= function (header) {
+function parse_params(header) {
   // This parser will definitely fail if there is more than one challenge
   var tok, last_tok, _i, _len, key, value;
   var state= 0;   //0: token,
-  m= header.split(Separators)
+  var m= header.split(Separators)
   for (_i = 0, _len = m.length; _i < _len; _i++) {
     last_tok= tok;
     tok = m[_i];
