diff --git a/lib/parsers.js b/lib/parsers.js
index 3375f70ff7465318407a9ac04e95a865144d26ce..10030d17d08d0bdf3b7a81ff6ef2724615aff464 100644
--- a/lib/parsers.js
+++ b/lib/parsers.js
@@ -2,7 +2,7 @@ var ParseAuth= /(\w+)\s+(.*)/  // -> scheme, params
   , Separators= /([",=])/
   ;
 
-parse_params= function (header) {
+function parse_params(header) {
   // This parser will definitely fail if there is more than one challenge
   var tok, last_tok, _i, _len, key, value;
   var state= 0;   //0: token,
@@ -56,7 +56,6 @@ parse_params= function (header) {
         else {
           return 'Unexpected token ('+tok+') after '+value+'"';
         }
-        continue;
       case 9: // expect commma
         if (',' != tok) return 'Comma expected after '+value;
         state= 0;
